tests/authcrypt.log

```diff
@@ -0,0 +1,4 @@
+plaintext message: [0-9a-fA-F]*
+ciphertext: [0-9a-fA-F]*
+decrypted: [0-9a-fA-F]*

```

Comment:
I think this regular expression is a little too simple. This will match something like:
```
plaintext message: 
ciphertext: 
decrypted: 
```
Which is not correct output because there is no hash value. Also, we know that the example uses SHA256 so the output length is always the same. Is it possible to change the expression to match a fixed number of characters? Also, SHA256 does not have any randomness, I do not think the output is expected to change between runs, but perhaps you should check that, because then you do not really need a regexp but a fixed value.